npc/re/merchants/enchan_illusion_16_2.txt

```diff
@@ -613,3 +614,695 @@ OnProgress:
+		mes "You must have heard about the strange places that have sprung up here and there lately. Illusion Stones are only found in those places,";
+		next;
+		mes "[Illusion Enchanter]";
+		mes "and their atomic structure is unlike any mineral I've seen. *Blah Blah* ...Their chemical chains... *Blah Blah*";
+		next;
+		select("(Uh-oh, he's zoning out!)");
+		mes "[Illusion Enchanter]";
+		mes "Anyway, I can enchant equipment that is exchanged for or purchased with Illusion Stones. I call them the Illusion equipment.";
+		next;
+		select("(He has no sense when it comes to names.)");
+		mes "[Illusion Enchanter]";
+		mes "This equipment at least responds positively to Illusion Stones, or it has the potential to be enchanted successfully.";
+		next;
+		mes "[Illusion Enchanter]";
+		mes "Explore the worlds of illusions, collect Illusion Stones, and exchange them for equipment. I can enchant the equipment for you.";
+		next;
+		mes "[Illusion Enchanter]";
+		mes "This benefits both of us. I need Illusion Stones for my studies, and you need powerful equipment for your adventures.";
+		next;
+		mes "[Illusion Enchanter]";
+		mes "That said, my service is not for free. It requires a few Illusion Stones and Zeny.";
+		next;
+		mes "[Illusion Enchanter]";
+		mes "If you're interested, how'd you like to get a membership for my service? You can get discounts that way!";
+		next;
+		if (select( "Sure!", "Not interested." ) == 2) {
+			mes "[Illusion Enchanter]";
+			mes "Okay, but let me know if you change your mind.";
+			close;
+		}
+		mes "[Illusion Enchanter]";
+		mes "Alright, then I'll add you to my member list. I hope you'll have some Illusion equipment next time I see you.";
+		setquest 12415;
+		completequest 12415;
+		close;
+	}
+	mes "[Illusion Enchanter]";
+	mes "I can enchant Illusion equipment.";
+	next;
+	switch( select( "How much do you charge?", "Add stats to Illusion equipment.", "Reset enchanted Illusion equipment." ) ) {
+	case 1:
+		mes "[Illusion Enchanter]";
+		mes "First, let me tell you that I can only enchant certain types of Illusion equipment.";
+		next;
+		mes "[Illusion Enchanter]";
+		mes "Up to 2 sockets can be enchanted, and enchanting each socket requires 5 Illusion Stones. You can reset enchanted Illusion equipment at any time.";
+		next;
+		mes "[Illusion Enchanter]";
+		mes "Enchanting never fails, but resetting sometimes failes. Even modern science can't change that.";
+		next;
+		mes "[Illusion Enchanter]";
+		mes "Resetting requires a different amount of Zeny depending on the success chance you choose. Select the menu if you'd like to see the available payment options.";
+		close;
+	case 2:
+		.@reset = false;
+		break;
+	case 3:
+		.@reset = true;
+		break;
+	}
+	setarray .@location[0], EQI_HAND_R, EQI_ARMOR, EQI_SHOES, EQI_GARMENT, EQI_ACC_R, EQI_HEAD_TOP, EQI_HAND_L;
+	.@s = select( "Right-handed Weapon", "Armor", "Shoes", "Garment", "Right Accessory", "Helm (Upper)", "Left-handed Weapon/Shield" ) - 1;
```

Comment:
Could you check if kRO doesn't have "Left Accessory" in the menu? because in the bRO it shows.
![image](https://user-images.githubusercontent.com/98105181/210280089-c91a8a50-d28d-4899-b9a7-e13d80302dc0.png)

Reply:
The missing menu has been added, thanks!